pom.xml

```diff
@@ -153,25 +152,13 @@
             <artifactId>commons-io</artifactId>
             <version>2.7</version>
         </dependency>
-        <dependency>
-            <groupId>javax.annotation</groupId>
-            <artifactId>javax.annotation-api</artifactId>
-            <version>1.3.1</version>
-        </dependency>
 
         <dependency>
             <groupId>com.oracle.database.jdbc</groupId>
             <artifactId>ojdbc10</artifactId>
             <version>19.11.0.0</version>
         </dependency>
 
-        <dependency>
-            <groupId>com.oracle.database.nls</groupId>
```

Comment:
Don't remove. It used by non-latin user of oracle, for example in Chinese encoding.